app/views/notifications/index.html.haml

```diff
@@ -1,8 +1,19 @@
-%article
+- content_for :title do
+  = 'Notifications'
+
+%article.user{ data: "settings"}
   %header
-    %h1 Notifications
-    %h2 What's been going on in GlitterGallery
-  %content
+    %h1
+      = avatar current_user.email
+      = 'Notifications'
+    %h2
+      = '@' + current_user.username
+
+  - @user = current_user
```

Comment:
We shouldn't be defining instance variables in the view - that's a job for the controller.